fight.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Fight.h"
+#include <iostream>
+Fight::Fight() {}
+Fight::Fight(int amount){
+    lenghtOfArray = amount;
+    arrayoffighters = new Fighter[amount];
+}
+Fight::~Fight() {
+    delete [] arrayoffighters;
+}
+void Fight::addFighterToArray(Fighter fighter,unsigned int INDEX){
+    if(INDEX<lenghtOfArray){
+        arrayoffighters[INDEX] = fighter;
```

Comment:
Harsh variable name. Please, rename from "**arrayoffighters**" to "**arrayOfFighters**".